### PR TITLE
 Fix (Updates): Inconsistencies

### PR DESCRIPTION
**Pull request by OjusWiZard**

### CODE REVIEW

File: updates/25/updates.sql [hunk 1]
```diff
@@ -1,4 +1,808 @@
-UPDATE assets SET identifier="eip155:56/erc20:0x201bC9F242f74C47Bbd898a5DC99cDCD81A21943" WHERE identifier="eip155:1/erc20:0x201bC9F242f74C47Bbd898a5DC99cDCD81A21943";
-INSERT INTO assets(identifier, name, type) VALUES("eip155:56/erc20:0x201bC9F242f74C47Bbd898a5DC99cDCD81A21943", "IGU Token", "C"); INSERT INTO evm_tokens(identifier, token_kind, chain, address, decimals, protocol) VALUES("eip155:56/erc20:0x201bC9F242f74C47Bbd898a5DC99cDCD81A21943", "A", 56, "0x201bC9F242f74C47Bbd898a5DC99cDCD81A21943", 18, NULL); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES("eip155:56/erc20:0x201bC9F242f74C47Bbd898a5DC99cDCD81A21943", "IGU", "iguverse-igu", "IGU", NULL, 1667125109, NULL);
```
Comment (yabirgb) [inline updates/25/updates.sql:2]: why remove those? If they didn't get applied before you need to keep them

File: updates/25/updates.sql [hunk 2]
```diff
@@ -1,4 +1,808 @@
...
-UPDATE evm_tokens SET chain=100 WHERE identifier="eip155:100/erc20:0x4d18815D14fe5c3304e87B3FA18318baa5c23820";
-INSERT INTO assets(identifier, name, type) VALUES("eip155:100/erc20:0x4d18815D14fe5c3304e87B3FA18318baa5c23820", "Safe", "C"); INSERT INTO evm_tokens(identifier, token_kind, chain, address, decimals, protocol) VALUES("eip155:100/erc20:0x4d18815D14fe5c3304e87B3FA18318baa5c23820", "A", 100, "0x4d18815D14fe5c3304e87B3FA18318baa5c23820", 18, ""); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES("eip155:100/erc20:0x4d18815D14fe5c3304e87B3FA18318baa5c23820", "SAFE", "safe", "safe", NULL, 1697181885, NULL);
```
Comment (yabirgb) [inline updates/25/updates.sql:4]: why remove this fix?

File: updates/25/updates.sql [hunk 3]
```diff
@@ -1,4 +1,808 @@
...
+INSERT INTO assets(identifier, name, type) VALUES("eip155:42161/erc20:0x23A941036Ae778Ac51Ab04CEa08Ed6e2FE103614", "Graph Token", "C"); INSERT INTO evm_tokens(identifier, token_kind, chain, address, decimals, protocol) VALUES("eip155:42161/erc20:0x23A941036Ae778Ac51Ab04CEa08Ed6e2FE103614", "A", 42161, "0x23A941036Ae778Ac51Ab04CEa08Ed6e2FE103614", 18, ""); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES("eip155:42161/erc20:0x23A941036Ae778Ac51Ab04CEa08Ed6e2FE103614", "GRT", "the-graph", "grt", NULL, 1623867469, NULL);
+INSERT INTO assets(identifier, name, type) VALUES("eip155:1/erc20:0x7b35Ce522CB72e4077BaeB96Cb923A5529764a00", "Impermax", "C"); INSERT INTO evm_tokens(identifier, token_kind, chain, address, decimals, protocol) VALUES("eip155:1/erc20:0x7b35Ce522CB72e4077BaeB96Cb923A5529764a00", "A", 1, "0x7b35Ce522CB72e4077BaeB96Cb923A5529764a00", 18, ""); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES("eip155:1/erc20:0x7b35Ce522CB72e4077BaeB96Cb923A5529764a00", "IMX", "impermax", "", NULL, 1619538925, NULL);
```
Comment (yabirgb) [inline updates/25/updates.sql:3]: why does insert? Are those the missing from the pools?

File: updates/22/updates.sql [hunk 4]
```diff
@@ -287,4 +287,6 @@ INSERT INTO assets(identifier, name, type) VALUES("RENDER", "Render Token", "Y")
...
 *
 INSERT INTO assets(identifier, name, type) VALUES("eip155:56/erc20:0xC5d27F27F08D1FD1E3EbBAa50b3442e6c0D50439", "Moon App", "C"); INSERT INTO evm_tokens(identifier, token_kind, chain, address, decimals, protocol) VALUES("eip155:56/erc20:0xC5d27F27F08D1FD1E3EbBAa50b3442e6c0D50439", "A", 56, "0xC5d27F27F08D1FD1E3EbBAa50b3442e6c0D50439", 18, NULL); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES("eip155:56/erc20:0xC5d27F27F08D1FD1E3EbBAa50b3442e6c0D50439", "APP", "moon-app", "", NULL, NULL, NULL);
+*
+INSERT INTO assets(identifier, name, type) VALUES("eip155:10/erc20:0x50c5725949A6F0c72E6C4a641F24049A917DB0Cb", "Lyra Token", "C"); INSERT INTO evm_tokens(identifier, token_kind, chain, address, decimals, protocol) VALUES("eip155:10/erc20:0x50c5725949A6F0c72E6C4a641F24049A917DB0Cb", "A", 10, "0x50c5725949A6F0c72E6C4a641F24049A917DB0Cb", 18, ""); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES("eip155:10/erc20:0x50c5725949A6F0c72E6C4a641F24049A917DB0Cb", "LYRA", "lyra-finance", "lyra", NULL, 1716186991, NULL);
```
Comment (yabirgb) [inline updates/22/updates.sql:291]: why adding this one here and not in 25?

File: updates/22/asset_collections_mappings_updates.sql [hunk 5]
```diff
@@ -54,8 +54,6 @@ INSERT INTO multiasset_mappings(collection_id, asset) VALUES (248, "eip155:1/erc
 *
 INSERT INTO multiasset_mappings(collection_id, asset) VALUES (248, "eip155:100/erc20:0x1509706a6c66CA549ff0cB464de88231DDBe213B");
 *
-INSERT INTO multiasset_mappings(collection_id, asset) VALUES (23, "eip155:10/erc20:0x50c5725949A6F0c72E6C4a641F24049A917DB0Cb");
-*
```
Comment (yabirgb) [inline updates/22/asset_collections_mappings_updates.sql:58]: have you created a note to remove this one i nthe db upgrade? We will forget otherwise.

Comment (OjusWiZard) [reply updates/22/asset_collections_mappings_updates.sql:58]: created here https://github.com/orgs/rotki/projects/11/views/2?pane=issue&itemId=63714158

File: updates/25/updates.sql (hunk 3 above)
Comment (OjusWiZard) [reply updates/25/updates.sql:3]: it's for a different asset. I've restored the removed one now

File: updates/25/updates.sql [hunk 6]
```diff
@@ -2,3 +2,813 @@ UPDATE assets SET identifier="eip155:56/erc20:0x201bC9F242f74C47Bbd898a5DC99cDCD
...
+*
+INSERT INTO assets(identifier, name, type) VALUES("eip155:8453/erc20:0x94025780a1aB58868D9B2dBBB775f44b32e8E6e5", "BetSwirl v2", "C"); INSERT INTO evm_tokens(identifier, token_kind, chain, address, decimals, protocol) VALUES("eip155:8453/erc20:0x94025780a1aB58868D9B2dBBB775f44b32e8E6e5", "A", 8453, "0x94025780a1aB58868D9B2dBBB775f44b32e8E6e5", 18, ""); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES("eip155:8453/erc20:0x94025780a1aB58868D9B2dBBB775f44b32e8E6e5", "BETS", "", "", NULL, 1705050881, NULL); 
+*
+INSERT INTO assets(identifier, name, type) VALUES("eip155:42161/erc20:0x9623063377AD1B27544C965cCd7342f7EA7e88C7", "Graph Token", "C"); INSERT INTO evm_tokens(identifier, token_kind, chain, address, decimals, protocol) VALUES("eip155:42161/erc20:0x9623063377AD1B27544C965cCd7342f7EA7e88C7", "A", 42161, "0x9623063377AD1B27544C965cCd7342f7EA7e88C7", 18, ""); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES("eip155:42161/erc20:0x9623063377AD1B27544C965cCd7342f7EA7e88C7", "GRT", "the-graph", "", NULL, 1669815527, NULL);
```
Comment (yabirgb) [inline updates/25/updates.sql:783]: add cc/coingecko for this

File: updates/25/updates.sql [hunk 7]
```diff
@@ -2,3 +2,813 @@ UPDATE assets SET identifier="eip155:56/erc20:0x201bC9F242f74C47Bbd898a5DC99cDCD
...
+*
+INSERT INTO assets(identifier, name, type) VALUES("eip155:8453/erc20:0x2D42dFF7323F585A0168e904a7c982B511c3B5A3", "Hatercoin", "C"); INSERT INTO evm_tokens(identifier, token_kind, chain, address, decimals, protocol) VALUES("eip155:8453/erc20:0x2D42dFF7323F585A0168e904a7c982B511c3B5A3", "A", 8453, "0x2D42dFF7323F585A0168e904a7c982B511c3B5A3", 9, ""); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES("eip155:8453/erc20:0x2D42dFF7323F585A0168e904a7c982B511c3B5A3", "HATER", "", "", NULL, 1695617299, NULL); 
+*
+INSERT INTO assets(identifier, name, type) VALUES("eip155:8453/erc20:0x6c8b42a354f97C4Ef09E97200f6f86DC41Bb02dB", "Curve DAO", "C"); INSERT INTO evm_tokens(identifier, token_kind, chain, address, decimals, protocol) VALUES("eip155:8453/erc20:0x6c8b42a354f97C4Ef09E97200f6f86DC41Bb02dB", "A", 8453, "0x6c8b42a354f97C4Ef09E97200f6f86DC41Bb02dB", 14, ""); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES("eip155:8453/erc20:0x6c8b42a354f97C4Ef09E97200f6f86DC41Bb02dB", "CRV", "", "", NULL, 1691152887, NULL);
```
[the remark this answers is not in the document]
Comment (OjusWiZard) [reply updates/25/updates.sql:619]: this seems like a spam asset, removing it. This `0x8Ee73c484A26e0A5df2Ee2a4960B789967dd0415` is the real one according to docs

**Review by yabirgb — APPROVED** [review]

**Pull request merged**